src/app.ts

```diff
@@ -716,3 +732,11 @@ export function exportApproved() {
   clearApprovedData();
   writeApprovedData(outputHeader, rowsToWrite);
 }
+
+function _testGenerateNextRow() {
```

Comment:
Should this be a proper unit test?

Reply:
no, it's a util for online debugging

Reply:
But SHOULD we instead make it a proper unit test? 😃
Would be helpful for automation.

Reply:
turned into debug function